tests/com/goide/inspections/GoHighlightingTest.java

```diff
@@ -295,18 +296,6 @@ public void testMainInMainPackage() {
     myFixture.checkHighlighting();
   }
 
-  public void testDuplicateBuiltinFunction() {
```

Comment:
I've removed the two separate tests and created a file for it, I don't know why I haven't done that the first time around.